BBBBroadcast/Services/WebRTCClient.swift

```diff
@@ -127,7 +128,7 @@ final class WebRTCClient: NSObject {
     private func createVideoTrack() -> RTCVideoTrack {
         videoSource = WebRTCClient.factory.videoSource(forScreenCast: true)
         videoCapturer = RTCVideoCapturer(delegate: videoSource!)
-        videoSource!.adaptOutputFormat(toWidth: 441, height: 736, fps: 15)
+        videoSource!.adaptOutputFormat(toWidth: 600, height: 800, fps: 15)
```

Comment:
Does this work for portrait and landscape?

Reply:
I uploaded video which demonstrates behaviour when switching between landscape and portrait to channel C27937 BigBlueButton iOS App in Teams.  
I think this need to be fixed on the BBB side, because I believe their video screen need to adapt. Not sure if they support switching between portrait and landscape mode. We will discuss this with Tiago.